src/features/homepage/Homepage.jsx

```diff
@@ -1,4 +1,45 @@
+import { Form, redirect, useLoaderData } from 'react-router-dom';
 import styles from './Homepage.module.css';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { searchoptionsLoaded } from '../search/searchSlice';
 export default function Homepage() {
-  return <div></div>;
+  const allBreeds = useLoaderData();
+  const dispatch = useDispatch();
+
+  // putting the breeds in the store
+  useEffect(
+    function () {
+      dispatch(searchoptionsLoaded(allBreeds));
+    },
+    [allBreeds, dispatch]
+  );
+
+  return (
+    <div className={`${styles.homepage}`}>
+      <Form method="POST">
```

Comment:
change the form so that the user can start typing and the options available will be filtered and shown to him and remove the submit button